VideoLocker/src/test/java/org/edx/mobile/test/ServerJsonDateAdapterFactoryTest.java

```diff
@@ -0,0 +1,73 @@
+package org.edx.mobile.test;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+
+import org.edx.mobile.core.ServerJsonDateAdapterFactory;
+import org.edx.mobile.util.DateUtil;
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public class ServerJsonDateAdapterFactoryTest {
+
+    private Gson newGson() {
```

Comment:
Since every test uses this object, you could just assign it to a field in a `@Before` method, so you don't need to call it in every test.